testing/regress/ecl/genjoin.ecl

```diff
@@ -67,6 +76,9 @@ jrec xfmgrp(jrec l, DATASET(jrec) r, STRING lab) := TRANSFORM
     SELF.rstr := r[c].rstr;
     SELF.c := c;
     SELF.label := lab;
+    SELF.ml := MATCHED(l);
```

Comment:
all three test files are generated from genjointest.pl with different selections for test cases.